src/lsm/radix_buffer.zig

```diff
@@ -0,0 +1,4 @@
+pub const RadixBuffer = struct {
```

Comment:
Let's call it `ScratchMemory`? I think that better captures it. It is both a ScratchMemory and a RadixBuffer, because it is scratch memory used as a radix buffer. But fro the implementation itself ,we care about it being scratch memory. 

And

```
radix_buffer: ScratchMemory
```

I think reads best!